src/BenchmarksClient/Workers/WrkWorker.cs

```diff
@@ -381,5 +383,17 @@ private static void Log(string message)
             var time = DateTime.Now.ToString("hh:mm:ss.fff");
             Console.WriteLine($"[{time}] {message}");
         }
+
+        public Task DisposeAsync()
+        {
+            _process.Dispose();
+            _process = null;
+            return Task.CompletedTask;
+        }
+
+        public DateTime GetWhenLastJobFinished()
```

Comment:
What is this function for?

Reply:
This was from an older implementation. Not needed anymore